src/ansys/mapdl/core/mapdl_geometry.py

```diff
@@ -105,10 +138,10 @@ def __init__(self, mapdl):
         self._lines_cache = None
         self._log = self._mapdl._log
 
-    def _set_log_level(self, level):
+    def _set_log_level(self, level: str) -> None:
```

Comment:
the type expected here is DEBUG_LEVELS (need to import it from ansys.mapdl.core.mapdl):
```suggestion
    def _set_log_level(self, level: DEBUG_LEVELS) -> None:
```